test/e2e/app/main.go

```diff
@@ -177,9 +190,11 @@ func startLightClient(cfg *Config) error {
 
 	providers := rpcEndpoints(tmcfg.P2P.PersistentPeers)
 
+	// TODO: that can not work! Some arguments required by light.NewHTTPClient are missing!
 	c, err := light.NewHTTPClient(
```

Comment:
Arguments passed to the NewHTTPClient at that line seems to be completely out of order, should I fix this in this PR as well?

Reply:
well we need to remove the trust options, what else needs to be changed?